node/utils/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const CRON = {
+    id: '2943b02a-fd27-47e3-9fcc-9eb00e07e2a0',
+    authToken: '52843179-97f4-4e49-b716-1ecf58809838',
```

Comment:
this is kind of useless if it's added on a public github project 🤷‍♂️ 

A better approach would be to read it from settings

Reply:
done